linkerd/protocol/h2/src/main/scala/io/buoyant/linkerd/protocol/h2/grpc/GrpcClassifierConfig.scala

```diff
@@ -4,8 +4,12 @@ import com.twitter.finagle.buoyant.h2.service.H2Classifier
 import io.buoyant.linkerd.ResponseClassifierInitializer
 import io.buoyant.linkerd.protocol.h2._
 
-class NeverRetryableConfig extends H2ClassifierConfig {
-  override def mk: H2Classifier = GrpcClassifiers.NeverRetryable
+abstract class GrpcClassifierConfig extends H2ClassifierConfig {
+  var nonAccruableStatusCodes: Option[Set[Int]] = None
```

Comment:
consider adding a `def` that implements the defaulting so that you don't have to repeat it.

`def _successStatusCodes = nonAccruableStatusCodes.getOrElse(Set(Ok()))`